maquetado/setup/index.html

```diff
@@ -9,6 +9,9 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <div class="title">
+    <img class = "title-decoration" src="assets/triangle.svg" alt="Title Decoration">
```

Comment:
You can use a pseudo element for this. Like `::after` or `::before` we usually use this for decorative elements like this